### DISCUSSION TITLE
Error 404- New repository

### DISCUSSION THREAD

**Discussion by Siddharth1981** [community/community#22848] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, i have a new repository and my main Index.html file loads perfectly, until, i try clicking the inside links to hobbies and contact.<br>
<a href="https://siddharth1981.github.io/CV/" class="onebox" target="_blank" rel="nofollow noopener">https://siddharth1981.github.io/CV/</a></p>
<p>help to advice what could be the reason.</p>
<p>Have checked everything to my understanding. Using safari, chrome and even incognito mode. but nothing works.</p>
<p>help<br>
sid.</p>

**Comment by thinkverse** [comment, marked as answer]
<p>Hi there. 👋</p>
<p>Your issue is two-fold, first GitHub pages is case sensitive, so when you go to <code>contact-me.html</code> GitHub will look for that file, that file doesn’t exist since it needs to have an uppercase <code>C</code> like the file.</p>
<p>So you can either change the file to be all lowercase or update the link to point to the correct file name, in this case, <a href="https://siddharth1981.github.io/CV/Contact-me.html" rel="nofollow noopener">Contact-me.html</a></p>
<p>For your second link, <code>hobbies.html</code> that file doesn’t exist in your <a href="https://github.com/Siddharth1981/CV" rel="nofollow noopener">repository</a>, so GitHub won’t know which file to load since it doesn’t exist, hence the 404 not found.</p>
<p>Hope that helped. :slightly_smiling_face:</p>

**Comment by Siddharth1981** [comment]
<p>Hi,</p>
<p>Appreciate your help and it all works fine. :+1:t3:</p>
<p>Thanks<br>
Sid</p>

**Comment by burbigo2** [comment]
<p>Thank you so much for your help, it’s working for me too</p>
